rlclientlib/vw_model/safe_vw.cc

```diff
@@ -318,6 +367,24 @@ bool safe_vw::is_CB_to_CCB_model_upgrade(const std::string& args) const
   return local_model_type == mm::model_type_t::CCB && inbound_model_type == mm::model_type_t::CB;
 }
 
+const char* safe_vw::get_audit_data() const
+{
+  if (_vw->audit) { return &(*(_vw->audit_buffer))[0]; }
```

Comment:
I am not certain that `audit_buffer` is a null terminated string. Would be good to either ensure or assert. Better yet this function could return a string_view as you definitely know the size

I think you can do `_vw->audit_buffer.data()`